be/src/runtime/sender_queue.cpp

```diff
@@ -545,6 +547,11 @@ void DataStreamRecvr::PipelineSenderQueue::decrement_senders(int be_number) {
     VLOG_FILE << "decremented senders: fragment_instance_id=" << print_id(_recvr->fragment_instance_id())
               << " node_id=" << _recvr->dest_node_id() << " #senders=" << _num_remaining_senders
               << " be_number=" << be_number;
+    if (UNLIKELY(_num_remaining_senders < 0)) {
```

Comment:
in which case _num_remaining_senders can be less than 0?

Reply:
when decrement sender from un-related BE, of other unexpected cases.